### PR TITLE
Port Java library

### PR DESCRIPTION
**Pull request by stefano81**

### CODE REVIEW

**Review by marcosimioni — APPROVED** [review]
This is great. LGTM! Just dropped some minor comments.

File: library/build.gradle [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        exceptionFormat 'short'
+
+        showCauses true
+        showExceptions true
+        showStackTraces true
+        showStandardStreams false
+    }
+}
+
+group = 'com.ibm.research.drl.dpt'
```
[the remark this answers is not in the document]
Comment (stefano81) [reply library/build.gradle:98]: Thank you for pointing it out. Change pushed to the `documentation` branch, to be reviewed.

File: library/build.gradle [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    implementation "ca.uhn.hapi:hapi-structures-v23:2.3"
+    implementation("org.dcm4che:dcm4che-imageio:5.10.6") {
+        exclude module: 'junit'
+        exclude module: 'slf4j-log4j12'
+    }
+    implementation "org.apache.commons:commons-text:${commons_text_version}"
+    implementation 'org.iban4j:iban4j:3.2.4-RELEASE'
+
+    testImplementation "org.apache.commons:commons-text:${commons_text_version}"
+    testImplementation 'org.postgresql:postgresql:42.5.1'
```
[the remark this answers is not in the document]
Comment (stefano81) [reply library/build.gradle:62]: Fair point. Adding issue for that.

File: library/src/main/java/com/ibm/research/drl/schema/IPVSchema.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
+/*******************************************************************
+ * IBM Confidential                                                *
+ *                                                                 *
+ * Copyright IBM Corp. 2015                                        *
+ *                                                                 *
+ * The source code for this program is not published or otherwise  *
+ * divested of its trade secrets, irrespective of what has         *
+ * been deposited with the U.S. Copyright Office.                  *
+ *******************************************************************/
+package com.ibm.research.drl.schema;
```
[the remark this answers is not in the document]
Comment (stefano81) [reply library/src/main/java/com/ibm/research/drl/schema/IPVSchema.java:6]: Adding issue.

File: library/src/main/java/com/ibm/research/drl/jsonpath/JSONPath.java [hunk 4]
```diff
@@ -0,0 +1,76 @@
+/*******************************************************************
+ *                                                                 *
+ * Copyright IBM Corp. 2016                                        *
+ *                                                                 *
+ *******************************************************************/
+package com.ibm.research.drl.jsonpath;
```
[the remark this answers is not in the document]
Comment (stefano81) [reply library/src/main/java/com/ibm/research/drl/jsonpath/JSONPath.java:6]: Added issue and closed

**Pull request merged**